frontend/components/dashboard/DashboardValidatorOverview.vue

```diff
@@ -1,18 +1,21 @@
 <script setup lang="ts">
 import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
 import { faArrowUpRightFromSquare } from '@fortawesome/pro-solid-svg-icons'
-import { useValidatorDashboardOverviewStore } from '~/stores/dashboard/useValidatorDashboardOverviewStore'
 import type { ClElValue } from '~/types/api/common'
 import {
   type NumberOrString, TimeFrames,
 } from '~/types/value'
 import { totalElClNumbers } from '~/utils/bigMath'
 import { DashboardValidatorSubsetModal } from '#components'
+import type { VDBOverviewData } from '~/types/api/validator_dashboard'
 
 const { t: $t } = useTranslation()
 const { converter } = useValue()
 
-const { overview } = useValidatorDashboardOverviewStore()
+const props = defineProps<{
+  data?: VDBOverviewData,
+}>()
+const { data } = toRefs(props)
```

Comment:
this should be spreaded out, so that this component will not have just one prop where the details are hidden